app/pubspec.yaml

```diff
@@ -48,6 +48,7 @@ dependencies:
   # pana version to be pinned
   pana: '0.22.20'
   # 3rd-party packages with pinned versions
+  bit_array: 2.3.0
```

Comment:
@jonasfj are we ok with this dependency?

Reply:
Note: the 80% of `package:bit_array` is probably not needed for `pub-dev`, and I have no issue with vendoring it.

Also: the package is using 32-bit ints for cross-compatibility in browsers, and it is possible we should benchmark using 64-bit ints for the server-only use case.